mkdocs/docs/verify-release.md

```diff
@@ -48,13 +48,18 @@ Set an environment variable to the version to verify and path to use
 
 ```sh
 export PYICEBERG_VERSION=<version> # e.g. 0.6.1rc3
+```
+
+And a temp folder for the artifacts
+
+```sh
 export PYICEBERG_VERIFICATION_DIR=/tmp/pyiceberg/${PYICEBERG_VERSION}
 ```
 
 Next, verify the `.asc` file.
 
 ```sh
-svn checkout https://dist.apache.org/repos/dist/dev/iceberg/pyiceberg-${PYICEBERG_VERSION}/ ${PYICEBERG_VERIFICATION_DIR}
+svn checkout https://dist.apache.org/repos/dist/dev/iceberg/pyiceberg-$PYICEBERG_VERSION/ ${PYICEBERG_VERIFICATION_DIR}
```

Comment:
previously `${PYICEBERG_VERSION}` would expand to `$\{PYICEBERG_VERSION\}/` in iterm and mess up the url